Frontend2/1.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="utility.css">
+    <link rel="stylesheet" href="main.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <title>ИКИТ СФУ</title>
+</head>
+<body class="page">
+    <article class="page__content content">
+        <header class="content__header header">
+            <img src="media/logo.png" class="header__image" alt="">
+            <div class="header__menu header-menu">
+                <a class="header-menu__item" href="#">Главная</a>
+                <a class="header-menu__item" href="#">Информация</a>
+                <a class="header-menu__item" href="#">Новости</a>
+                <a class="header-menu__item" href="#">Контакты</a>
+                <a class="header-menu__item" href="#">Главная</a>
+                <span class="header-menu__profile-item profile-menu-item">
+                    Артем
+                    <img src="media/profilepic.jpg" class="profile-menu-item__image" alt="">
+                </span>

```

Comment:
блоки можно по проще называть
```suggestion
                <span class="header__avatar avatar">
                    Артем
                    <img src="media/profilepic.jpg" class="avatar__image" alt="">
                </span>

```